mars/oscar/backends/pool.py

```diff
@@ -59,8 +59,9 @@
 
 
 class _ErrorProcessor:
-    def __init__(self, message_id: bytes, protocol):
+    def __init__(self, message_id: bytes, address: str, protocol):
```

Comment:
Simply use `pool` as the argument?

Reply:
Currently, the `_ErrorProcessor` needs some message info and the address of current pool. Replacing `address` to `pool` does not simplify the constructor parameters.